src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2642,3 +2654,73 @@ export function parseExitResult(
+		this._contentDomNode.setClassName('terminal-accessibility-help');
+		this._contentDomNode.setAttribute('role', 'document');
+		this._domNode.setDisplay('none');
+		this._domNode.setAttribute('role', 'dialog');
+		this._domNode.setAttribute('aria-hidden', 'true');
+		this._domNode.appendChild(this._contentDomNode);
+		this.onblur(this._contentDomNode.domNode, () => this.hide());
+		this._register(dom.addStandardDisposableListener(this._contentDomNode.domNode, 'keydown', (e) => {
+			if (e.keyCode === KeyCode.Escape) {
+				this.hide();
+			}
+		}));
+	}
+
+	public hide(): void {
+		if (!this._container) {
+			return;
+		}
+		this._domNode.setDisplay('none');
+		this._contentDomNode.setDisplay('none');
+		this._domNode.setAttribute('aria-hidden', 'true');
+		this._contentDomNode.domNode.tabIndex = -1;
+	}
+
+	attach(container: HTMLElement): void {
+		this._container = container;
+		this._domNode.setDisplay('block');
+		this._domNode.setAttribute('aria-hidden', 'false');
+		this._contentDomNode.domNode.tabIndex = 0;
+		this._buildContent();
+		container.appendChild(this._contentDomNode.domNode);
+		this._contentDomNode.domNode.focus();
+	}
+
+	private _buildContent(): void {
+		const runRecentKbLabel = this._keybindingService.lookupKeybinding('workbench.action.terminal.runRecentCommand')?.getAriaLabel();
+		const goToRecentKbLabel = this._keybindingService.lookupKeybinding('workbench.action.terminal.goToRecentDirectory')?.getAriaLabel();
+		const enterAccessibilityModeLabel = this._keybindingService.lookupKeybinding('workbench.action.terminal.enterAccessibilityMode')?.getAriaLabel();
+		const openDetectedLinkLabel = this._keybindingService.lookupKeybinding('workbench.action.terminal.openDetectedLink')?.getAriaLabel();
+		let content = nls.localize('introMsg', "Welcome to Terminal Accessibility Help\n\n");
+		content += strings.format(nls.localize('enterAccessibilityMode', 'The Enter Accessibility Mode ({0}) command enables screen readers to read the terminal buffer'), enterAccessibilityModeLabel);
+		content += '\n\n';
+		if (this._hasShellIntegration) {
+			content += nls.localize('shellIntegration', "The terminal has a feature called shell integration which offers an enhanced experience. \n\nIt provides some useful commands for screen readers such as  ");
+			content += strings.format(nls.localize('runRecentCommand', 'Run Recent Command ({0})'), runRecentKbLabel);
+			content += ' and ';
+			content += strings.format(nls.localize('goToRecentDirectory', 'Go to Recent Directory ({0})'), goToRecentKbLabel);
+			content += '\n\n';
+		}
+		content += strings.format(nls.localize('detectedLink', 'The Open Detected Link ({0}) command enables screen readers to easily open links found in the terminal'), openDetectedLinkLabel);
+		content += '\n\n';
+		content += nls.localize('minContrastRatio', "The setting terminal.integrated.minimumContrastRatio takes a number between 1 and 21 and will adjust the text color luminance to meet this\n\n");
```

Comment:
Let's not mention this as it's for sighted only, so it's likely not applicable for screen readers.